fs/io_zenfs.cc

```diff
@@ -636,6 +637,72 @@ size_t ZonedRandomAccessFile::GetUniqueId(char* id, size_t max_size) const {
   return zoneFile_->GetUniqueId(id, max_size);
 }
 
+ZenFSGCWorker::ZenFSGCWorker() {
+  
+  total_residue_ = 0;
+
+}
+
+void ZenFSGCWorker::CheckZoneValidResidualData() {
+
+  std::map<std::string, ZoneFile*>::iterator it;
+  fs->files_mtx_.lock();
+  for(it = fs->files_.begin(); it != fs->files_.end(); it++) {
+
+    ZoneFile* existFile;
+    existFile = it->second;
+
+    for(auto ext_it : existFile->extents_) {
+
+      ZoneExtent* extent;
+      extent = ext_it;
+      
+      Zone* zone_idx = extent->zone_;
+      //only care about the FULL zone.
+      if(!zone_idx->IsFull()) {
+        break;
```

Comment:
I have no background about this project, but I doubt this logic. If one zone of an extent is already full, shall we ignore all following extents? If I guess correctly, we should use `continue` here.

Reply:
It is my code, **continue** is correct so far,.
Zifeng would help to fix it.